addons/account/views/report_payment_receipt_templates.xml

```diff
@@ -66,7 +66,7 @@
                                 <tr>
                                     <td><span t-field="inv.invoice_date">2023-01-01</span></td>
                                     <td><span t-field="inv.name">INV001</span></td>
-                                    <td><span t-field="inv.ref">Sample Ref</span></td>
```

Comment:
![image](https://github.com/user-attachments/assets/2fb85fd8-7ec4-42d7-8c07-181b2dac9207)
@jco-odoo I guess most of the issues I found to be re-producible were from payment receipt